app/javascript/react/components/WeekView/WeeklyMockData.tsx

```diff
@@ -0,0 +1,27 @@
+const today = new Date();
+
+const getDayBefore = (before: number): Date => {
+  const today = new Date();
+  const upDate = today.setDate(today.getDate() - before);
+  return new Date(upDate);
+};
+
+const weeklyData = [
+  { value: 25, date: today },
+  { value: 90, date: getDayBefore(1) },
+  { value: 80, date: getDayBefore(2) },
+  { value: 2, date: getDayBefore(3) },
+  { value: 60, date: getDayBefore(4) },
+  { value: 200, date: getDayBefore(5) },
+  { value: 30, date: getDayBefore(6) },
+];
+
+const thresholdsValues = {
+  min: 0,
+  low: 40,
+  middle: 50,
+  high: 80,
+  max: 100,
+};
```

Comment:
This isn't needed, as we can use the values selected from the story at this point - it's done in Calendar Page rn. :)